src/server/routes/universal.js

```diff
@@ -4,9 +4,7 @@ import { universal, homepage } from '../handlers/universal';
 
 const router = Router();
 
-if (process.env.NODE_ENV !== 'development') {
```

Comment:
Sorry, I agree that this solution looks cleaner than doing it in inside of the route and calling `universal` from there.
Just change the name from `homepage` to `homepageRedirect` to make it more clear that "if not in dev env, redirect home page".

Reply:
Updated :+1: